.gitmodules

```diff
@@ -31,3 +31,6 @@
 [submodule "lib/sdl/SDL2_image"]
 	path = lib/sdl/SDL2_image
 	url = https://github.com/libsdl-org/SDL_image.git
+[submodule "lib/usb/ohci_usbhostlib"]
```

Comment:
*(This is a generic comment about PR description, but made on code so we can resolve it more easily)*

> The HID driver can properly distinguish between various Xinput devices (Duke, Steel Battalion, Xremote, Xbox360, Xbone etc)

Personally, I'm not a big fan of supporting non-original devices (like Xbox 360 and Xbox One) as part of core nxdk.
Feel free to ignore this feedback, but as a purist I think it's weird to support these.
I'm also slightly concerned with people adding PS4 gamepad drivers etc., leading to an explosion of complexity.
While I'd like those features *for some projects* myself, I don't think it's ideal for upstream inclusion in the nxdk core (which, I feel, should be a very minimalistic C toolchain.. ideally a package manager would bring these independent parts together).